src/components/login/container/LoginDetails.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../styles/index.css';
+import newVideo from '../styles/video/newInstructions.mp4';
+
+const LoginDetails = () => (
+  <div className="login-details">
+    <div className="card">
+      <div className="card-header">
+        <strong>What to ensure</strong>
+      </div>
+      <div className="card-body">
+        Ensure you have an account from the traditional OCL. After that,{' '}
+        <Link to="https://qa.openconceptlab.org/accounts/login/">
+          Login into the traditional OCL
+        </Link>
```

Comment:
This should be an `a` tag that opens a new tab so the users won't have to completely leave the app they are trying to learn how to use.

Reply:
Done it